portal-frontend/src/app/shared/owner-dialogs/parcel-owners/parcel-owners.component.ts

```diff
@@ -91,13 +91,11 @@ export class ParcelOwnersComponent {
         },
       });
     }
-    dialog.beforeClosed().subscribe((updatedUuid) => {
-      if (updatedUuid) {
+    dialog.afterClosed().subscribe((result) => {
+      if (result) {
         this.onOwnersUpdated.emit();
       }
-    });
-    dialog.afterClosed().subscribe((result) => {
-      if (result.ownerDeleted) {
```

Comment:
Do we still need this fix and special logic for delete? I thought the above change would fix it.

Reply:
no, removed it

Reply:
Can you also clean up
`this.dialogRef.close({ ownerDeleted: true, deletedOwnerId: this.existingUuid});`
Someone might see that logic, and assume the code to handle it was accidentally deleted and see it as a bug, rather than code that isn't used.

Reply:
yeah, makes sense, removed that logic too